R/tm_variable_browser.R

```diff
@@ -81,7 +81,8 @@
 #' @export
 #'
 tm_variable_browser <- function(label = "Variable Browser",
-                                datasets_selected = character(0),
+                                datasets_selected = "all",
+                                datanames = datasets_selected,
                                 parent_dataname = "ADSL",
```

Comment:
I think we can discuss `parent_dataname` later. It doesn't make so much sense as parent is a property of `teal_data`. Not for this PR